lib/bitmask_attributes/definition.rb

```diff
@@ -138,6 +138,12 @@ def #{attribute}?(*values)
               self.#{attribute}.present?
             end
           end
+
+          def #{attribute}_any?(*values)
```

Comment:
Why is this doing an array lookup as opposed to an actual bitmask `or` operation?